__tests__/src/getPropValue-babelparser-test.js

```diff
@@ -1054,7 +1054,7 @@ describe('getPropValue', () => {
     it('should return string representation of variable identifier wrapped in a Typescript non-null assertion', () => {
       const prop = extractProp('<div foo={bar!} />');
 
-      const expected = 'bar';
+      const expected = 'bar!';
```

Comment:
I'm concerned this might be a breaking change. What do you think?

Reply:
I'm honestly not sure, my knowledge is limited here. I'm just happy to contribute. I thought that's what you wanted, according to the comments, but may have misunderstood.  If you could guide me & describe your ideas on how it should be done, that would be great. Thanks.

Reply:
So, to be clear, I'm not 100% sure what would be best here (to include the `!`, or not to). Your test descriptions said the `!` was included, so I pointed out that the implementation didn't match that.

When I link this change into `eslint-plugin-jsx-a11y` and `eslint-plugin-react`, nothing fails, so that's something - maybe it's worth a shot.